app/views/users/show.html.erb

```diff
@@ -1 +1,43 @@
-<div>This is a single user</div>
\ No newline at end of file
+<div class="blist-group-item list-group-item-action list-group-item-light border card mb-2">
+     <div class="d-flex row">
+    <div class="d-flex col-md-2">
+        <img src=<%= @user.photo %> class="img-thumbnail rounded" alt="User Photo" />
+    </div>
+    <div class="d-flex align-items-center col-md-8">
+      <div class="card-body align-items-center">
+        <h2 class="card-title"><%= @user.name %></h2>
+        <p class="card-text d-flex justify-content-end "><small class="text-muted">Number of posts: <%= @user.posts_counter%></small></p>
+      </div>
```

Comment:
- [Optional] It is important to make out codes DRY by using Rails partials. As you can see, this piece of code is being repeatedly used.